Tests/ML/test_metrics.py

```diff
@@ -154,3 +159,68 @@ def test_get_column_name_for_logging() -> None:
     hue_name = "foo"
     assert f"{hue_name}/{expected_metric_name}" == \
            get_column_name_for_logging(metric_name=metric_name, hue_name=hue_name)
+
+
+
+def test_accuracy05():
+    metrics = Accuracy05()
+    output = [torch.tensor([0.9, 0.8, 0.6]), torch.tensor([0.3, 0.9, 0.4])]
+    label = [torch.tensor([1., 1., 0.]), torch.tensor([0., 0., 0.])]
+    n_batches = len(output)
+    for i in range(n_batches):
+        metrics.update(output[i], label[i])
+    result = metrics.compute()
+    assert result == 4.0 / 6
+
+def test_classification_metrics():
+    metrics = [AccuracyAtOptimalThreshold(), OptimalThreshold(),
```

Comment:
can you rely on _get_metrics_classes here, rather than re-defininig the list of metrics that you are using?